plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -224,6 +224,18 @@ private function get_feature_definitions() {
 					'is_legacy'          => true,
 					'is_experimental'    => false,
 				),
+				'site_visibility_badge' => array(
+					'name'               => __( 'Site visibility badge', 'woocommerce' ),
+					'description'        => __(
+						'Enable the site visibility badge in the WordPress admin bar',
+						'woocommerce'
+					),
+					'enabled_by_default' => true,
```

Comment:
@ilyasfoo I think I would like to have the same behavior for new vs existing sites. In hindsight it would have been good to roll the feature out like that. However now that it has been released to all sites, and we are seeing significant usage of the badge in live mode, I think it will be too unexpected for those users to then have it removed. 

I think it will be better to allow existing users to make their own choice. I think we'll need to do some extra efforts to reach out -- blog post, twitter -- to raise awareness about the new setting.